src/Components/Messages/MessageChatCard/MessageChatCard.jsx

```diff
@@ -1,11 +1,22 @@
-import React from "react";
+import Chatrooms from "../../Chatrooms/Chatrooms"
+import React,{ useEffect, useState } from "react";
 import "./mCard.css";
+
 const MessageChatCard = (props) => {
+  //const [room,setRoom]=useState(false)
+  console.log("chat prop:", props.chat)
+  console.log("username:", props.userName)
+  //console.log("GenreData:", props.allGenres)
   const chatIcon =
     props.chat.title.split("")[0].toUpperCase() +
     props.chat.title.split("")[1].toUpperCase();
+    const handleNavigation = () => {
+      console.log("Window open!!")
+      //setRoom(true)
+      window.open(`http://localhost:3000/chatrooms?room=${props.chat.title}`)
```

Comment:
I'm not sure I like how this opens a new tab. Is it possible to open the chatroom without opening a new tab?

Reply:
Yeah, I don't mind changing it. I'll just redirect to another route on click.